django-stubs/test/testcases.pyi

```diff
@@ -216,7 +218,11 @@ class TransactionTestCase(SimpleTestCase):
 class TestCase(TransactionTestCase):
     @classmethod
     def setUpTestData(cls) -> None: ...
-    def captureOnCommitCallbacks(cls, *, using: Optional[str] = ..., execute: bool = ...): ...
+    @classmethod
+    @contextmanager
+    def captureOnCommitCallbacks(
+        cls, *, using: Optional[str] = ..., execute: bool = ...
```

Comment:
`using` should never have been marked `Optional`, it's always a string, it just has a default (indicated by `= ...`)	

```suggestion
        cls, *, using: str = ..., execute: bool = ...
```